app/js/actions.js

```diff
@@ -32,10 +32,29 @@ export class LaunchApp extends Action {
 @action('login')
 @propInject('userGateway')
 export class Login extends Action {
-    exec({ credentials }) {
+    exec({ credentials, presenter }) {
         this.logger.log('Attempt to login w/ credentials', credentials);
+
+        const errors = this._validate(credentials);
```

Comment:
Well there really is no good use case anymore for var as far as I can see. At yuzu, I didnt use let because when we first used transpiler there was a lot of magic and crap to make it work, so I though it was not worth it. Any case, that might not be true, and for this project I'd like to adopt them as they are intended. So I'll use const when I do not plan on reassigning that object, and let if do.